pkg/rancher-desktop/backend/lima.ts

```diff
@@ -1627,6 +1633,9 @@ export default class LimaBackend extends events.EventEmitter implements VMBacken
           this.progressTracker.action('Configuring containerd', 50, this.configureContainerd()),
         ]);
 
+        if (config.containerEngine.imageAllowList.enabled) {
+          await this.startService('openresty');
```

Comment:
We should avoid conditionally starting services from code where it's easily avoidable, to handle cases like things being started outside our expected paths (… possibly because we just started them elsewhere).

Reply:
How is this different from conditionally starting containerd and docker? We always start openresty before we start the container engines, and always stop it afterwards. So if this works for the container engines, it should work for openresty as well.